src/main/groovy/edu/oregonstate/mist/locations/frontend/jsonapi/ResourceObject.groovy

```diff
@@ -0,0 +1,8 @@
+package edu.oregonstate.mist.locations.frontend.jsonapi
+
+class ResourceObject {
+    String id
+    String type
+    //@todo: how do we want to handle this???
+    def attributes
```

Comment:
Thanks for asking this question! I refreshed on def. Def is an alias of Object, but it has the benefit that allows us to use some of the dynamic typing. Since attributes can hold a variety of things depending on the API, I'm going to leave it as is.